wire-maven-plugin/build.gradle

```diff
@@ -1,4 +1,5 @@
 apply plugin: 'com.liferay.maven.plugin.builder'
+apply plugin: 'org.jetbrains.kotlin.jvm'
```

Comment:
Modules that depend on runtime module have to get built by a Kotlin plugin, even though they're Java libraries - this is needed so that they can resolve the correct configuration of a multiplatform project

Reply:
Or I can actually point them to correct configuration manually, similar to `:wire-runtime:japicmp`.